interface/fax/fax_dispatch.php

```diff
@@ -751,35 +750,35 @@ function allCheckboxes(issel) {
 
 <p><input type='checkbox' name='form_cb_forward' value='1'
  onclick='return divclick(this,"div_forward");' />
-<b><?php echo xlt('Forward Pages via Fax'); ?></b></p>
+<strong><?php echo xlt('Forward Pages via Fax'); ?></strong></p>
```

Comment:
The `<p>` element that wraps this should have the class of `font-weight-bold` and the `<strong>` element then can be removed.